test/Finder.js

```diff
@@ -0,0 +1,47 @@
+const { didContractThrow } = require("./utils/DidContractThrow.js");
+
+const Finder = artifacts.require("Finder");
+
+contract("Finder", function(accounts) {
+  const owner = accounts[0];
+  const writer = accounts[1];
+  const user = accounts[2];
+
+  // Corresponds to Finder.Roles.Writer;
+  const RolesEnumWriter = "1";
+
+  it("General methods", async function() {
+    const finder = await Finder.new({ from: owner });
+    await finder.resetMember(RolesEnumWriter, writer, { from: owner });
+
+    const implementationAddress1 = web3.utils.toChecksumAddress("0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d");
```

Comment:
Seems fine to me. In other cases, I might prefer having the same input every time you run the test case, in case you're debugging by running the test repeatedly.